test/kubemark/gce/util.sh

```diff
@@ -20,7 +20,7 @@
 function run-gcloud-compute-with-retries {
   RETRIES="${RETRIES:-3}"
   for attempt in $(seq 1 ${RETRIES}); do
-    local -r gcloud_result=$(gcloud compute "$@" 2>&1)
+    gcloud_result=$(gcloud compute "$@" 2>&1)  # We don't use 'local -r' here as then ret_val seems to always get value 0.
```

Comment:
Thanks for adding a comment.
Can you please move it to the previous line (to make the line shorter).